students/jemielity_kamil/lesson_03_functions/the_collatz_sequence_input_validation.py

```diff
@@ -0,0 +1,27 @@
+
+def collatz(number):
+    try:
+        if number % 2 == 0:
+            print(number // 2)
+            return number // 2
+        else:
+            print(3 * number + 1)
+            return 3 * number + 1
+    except ValueError:
+        print('Value rror')
```

Comment:
Please try to avoid so broad exception handling, it should concern only 
```
try:
    even = number % 2 == 0
except:
    print('Value error')
```